outputs.tf

```diff
@@ -74,6 +74,22 @@ output "cluster_names" {
   ]
 }
 
+output "workload_cluster_id" {
```

Comment:
This is not going to work, since "workload" and "management" are not fixed VPC names. We actually have an input where the consumer can specify any number of VPCs, and their name. It defaults to ["workload","management"] but we expose this at catalog level as well.

The outputs might have to be simply "cluster_ids" and be some type of map or object where the name and ID are output, and you cannot assume how many or what their names are.

Reply:
For reference, this is the input I was talking about, this is exposed to the user. They can change the names and the number of VPCs, and this is also used when deploying the clusters: https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/blob/main/patterns/roks/variables.tf#L45

Reply:
https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/issues/705
The issue specifically asks to output the workload and management cluster ids and NULL if anything other is passed.

Reply:
also, if the consumer had used override, the name might not exactly match "-workload-cluster" (or management-cluster). I don't know if this is an issue, might want to check.